pretix_mollie/views.py

```diff
@@ -212,8 +212,28 @@ def _set_settings(ev):
     )
 
 
+def get_or_create_payment(payment, mollie_id, data):
+    if payment.info_data.get("id") != mollie_id:
+        for op in OrderPayment.objects.filter(order=payment.order):
+            if payment.info_data.get("id") == mollie_id:
+                payment = op
+                break
+        else:
+            payment = OrderPayment(
+                order=payment.order,
+                amount=Decimal(data.get("amount", {}).get("value", 0)),
+                provider=f'mollie_{data.get("method")}',
```

Comment:
I'm a little worried that some day we'll have a situation where our method name differs from the mollie one. Do we really expect method changes here or could we fix it to payment.provider?

Reply:
As of right now, I have only witnessed duplicate transactions of the same type. I guess for now we can keep that assumption and worry about it, once we see different hings happening